### DISCUSSION TITLE
Can we have a discord server?

### DISCUSSION THREAD

**Discussion by txtyash** [helix-editor/helix#859] [General] [answered]
Everyone just uses discord and it is really convenient. People already have a discord account and are familiar with it. I bet Discord server will have more members than matrix. Discord would be much better to have since discussions for this editor seem like a must as there's not much in the documentation, no tutorials on youtube, so discussion would be of help for beginners who would later help other beginners and y'all would have less worries as there'd be less github issues and it'll also attract more audience.

**Comment by archseer** [comment, marked as answer]
I don't find Discord (or Slack or any other walled garden for that matter) a good choice for software projects ([for example](https://sneak.berlin/20200220/discord-is-not-an-acceptable-choice-for-free-software-projects/)).

Matrix on the other hand is just a protocol and a spiritual successor to IRC, meaning we can always host our own node if needed. If you want a Discord-like UI you can always use [Cinny](https://cinny.in/), but Matrix also owns [Gitter](https://gitter.im/) so it should be possible to access it through there as well.

**Reply by txtyash** [reply]
But at the same time you use github instead of gitlab?

Yea, discord can be seen as a spyware but is it that big of a deal?

**Reply by archseer** [reply]
How is gitlab different from github?

Using git doesn't lock us into a single vendor, and we can always leave github in the future for SourceHut or use our own hosted git repositories (like kernel.org does) without any big difficulties.

> can be seen as a spyware

For me it's more about not using proprietary tools to develop an open source project, I find it unreasonable and uninclusive. I have nothing against Discord (I also use it), it's a great product but it's not a great fit for us/open source (Imagine they closed down or started charging? With Matrix we can host our own again).

**Reply by txtyash** [reply]
gitlab is opensource, github isn't.

github is spyware too. 

BTW, I understand your concerns about discord. It makes sense to me now.
Thanks.

**Comment by nico-abram** [comment]
[zulip](zulipchat.com) is another alternative.

As an example, most official rustc/rust-language communication that's not on github or discourse happens there (https://rust-lang.zulipchat.com/), although they do have an official discord and my understanding is some teams prefer it.

**Comment by theunrealtarik** [comment]
can we just have a discord server?

**Reply by ghost** [reply]
I have created one here if people want to join! https://discord.gg/zH8QBjBB

I love Helix and would love to start a community of people who love it as well!

We can learn from each other!

**Comment by ghost** [comment]
Here is a community run (not affiliated with the Helix team but would love to collaborate)discord server for Helix (just getting started so join the conversation)

https://discord.gg/GFmAn28d

**Reply by Axlefublr** [reply]
noooo the link is invalid :c can you send a new one? I want to join!

**Reply by Axlefublr** [reply]
iirc by default, an invite link only lasts for a week; you need to change that in your server's settings, or something like that

**Reply by ghost** [reply]
Here is a new link the expiration date is set to never expire. I am looking for admins too BTW.

https://discord.gg/TmhHeqSWav
